### DISCUSSION TITLE
🧑‍🎨 Logo redesign

### DISCUSSION THREAD

**Discussion by N3v1** [TeamCodeArena/DeliverEase#113] [Ideas]
Hi @SID262000, @neelansh27, @Devansh-Singh-Parmar, @Atrika10, @AryanGupta0007, everyone,
I'm currently working on a new logo and I showed @SID262000 to images. He likes this the most:

<img width="128" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/8f09022e-bab7-4545-927b-dc8fff72f8d7">

> **Note** \
> You might have to enable light mode to see it

As per @SID262000's request, can you suggest any additional colors for this logo?

<hr>

> **Note** \
>I would suggest using either a linear gradient or a solid color.

**Comment by Devansh-Singh-Parmar** [comment]
Please give some other choices.

**Reply by N3v1** [reply]
Okay @Devansh-Singh-Parmar

**Reply by SID262000** [reply]
@Devansh-Singh-Parmar In my view, we should wait for other view points as well before diverting @N3v1 for additional efforts.

**Reply by N3v1** [reply]
[**Here**](https://www.figma.com/file/KXQwarryqUevI0SOjiRASF/Deliver-Ease-Logo?type=design&node-id=0%3A1&mode=design&t=OuMIeBvoiwXO89Sa-1
) is the Figma file. Logo A needs some improvement - it's just a sketch

**Reply by SID262000** [reply]
With regards to A, is it just a low fidelity wireframe prototype that needs to be converted to a high fidelity one. Can you please confirm.
cc: @N3v1

**Reply by N3v1** [reply]
Yes @SID262000, If you want prototype A I have to design it more - like i said it's just a sketch

**Reply by Devansh-Singh-Parmar** [reply]
okay waiting for others, till then I will try to create one too.
cc @N3v1 @SID262000

**Reply by N3v1** [reply]
@Devansh-Singh-Parmar, You can create a new Frame in Figma in the file I shared with you and then create your logo. If you need any help, just let me know

cc: @SID262000

**Reply by SID262000** [reply]
@Devansh-Singh-Parmar Please don't divert yourself into this Issue, we should have clear goals set for each team member considering the shortage of time, continue working on the mainline issues.
@N3v1 shall continue working on this Issue solely to the best of his judgement as a primary contributor, as always we're all there for any assistance required, Thanks!

**Reply by KartikKar19** [reply]
@N3v1 The logo looks good we can also try a radial gradient to show a contrasting effect or we can just add some white lines underneath the truck showing that it is moving.

**Reply by N3v1** [reply]
Sure, @KartikKar19. Which logo you mean A or B?

**Reply by KartikKar19** [reply]
@N3v1 its about logo A

**Reply by N3v1** [reply]
@KartikKar19, could you please clarify your request? Would you like me to create a sketch to help illustrate what you mean?

**Reply by KartikKar19** [reply]
@N3v1 I was saying to just try adding three white lines underneath the truck so that it looks like it is moving. If it looks good you can keep them else remove the lines

**Reply by N3v1** [reply]
Okay @KartikKar19 I'll create a new frame for that

**Reply by SID262000** [reply]
Except @N3v1 ,
Everyone is requested to focus on upcoming development issues or any open issues in the rack. Don't divert yourself into unnecessary items in the last cycle. Thanks

**Comment by Devansh-Singh-Parmar** [comment]
I am not involved iinto any issue right now, I can try if needed.
cc @N3v1 @SID262000

**Reply by N3v1** [reply]
If you desire to do so, please go ahead @Devansh-Singh-Parmar

**Comment by neelansh27** [comment]
@N3v1 I think it looks good but we can try changing the background to solid orange or orange gradient but if others are fine with this logo then I guess there's no need to change it. 
Thank you

**Reply by N3v1** [reply]
Which logo do you mean? A or B? @neelansh27
